pages/releases/4.3.html

```diff
@@ -0,0 +1,2313 @@
+			<div class="container">
+				<h3>
+					<a href="#gdextension">GDExtension</a>
+					<div class="section-robot godot-icon-godot" aria-disabled="true"></div>
+				</h3>
+			</div>
+		</div>
+		<div class="container">
+			<div class="release-cards">
+				<div id="gdextension-runtime-class-registration" class="release-card span-3 media-right">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdextension-runtime-class-registration">
+										Register runtime classes with GDExtension
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										This feature is particulary interesting for those of you that use GDExtensions
+										to code game logic.
+									</p>
+									<p>
+										Before, whenever you wanted to register a GDextension class that is only used
+										during runtime, you had to wrap it in a check to see if you were currently in
+										the editor or not
+										(<code class="highlight"><span class="function">is_editor_hint</span><span class="symbol">()</span></code>).
+									</p>
+									<p>
+										How you declare a runtime class depends on the bindings you use. For
+										the officially maintained
+										<a href="https://github.com/godotengine/godot-cpp/">godot-cpp</a>
+										you use <code class="highlight"><span class="function">GDREGISTER_RUNTIME_CLASS</span><span class="symbol">(</span>ExampleRuntime<span class="symbol">)</span></code> in your module
+										initialization; screenshot on the right for reference.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="dsnopek"
+									data-read-more="https://github.com/godotengine/godot/pull/82554">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<img class="release-card-image position-bottom-left"
+								src="/storage/releases/4.3/images/gdextension-runtime-class.webp"
+								alt="Image of the documentation syntax highlight and copy button features." draggable="false" />
+						</div>
+					</div>
+				</div> <!-- #gdextension-runtime-class-registration -->
+
+				<div id="gdextension-register-virtual-methods" class="release-card span-2 inverted">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdextension-register-virtual-methods">
+										Register virtual methods
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										A virtual method (also known as virtual function) is a declared within a
+										base class and has a default implementation. Opposed to abstract methods,
+										which don't have actual code in them, you are not obligated to override
+										virtual methods when you derive a class.
```

Comment:
I'm not sure about this explanation.

Godot's virtual methods (meaning those registered in `ClassDB`, not the C++ ones) are a little different than normal virtual methods. First of all, they can't have default implementations, although the thing calling them can do something by default if not implemented. Also, you may in fact be obligated to implement a virtual method, it depends on if it's called via `GDVIRTUAL_CALL()` or `GDVIRTUAL_REQUIRED_CALL()`.

Maybe go for something simpler and less detailed like:

```suggestion
										A virtual method (also known as virtual function) is declared within a
										base class, and child classes or attached scripts can provide their own implementation, which is called by the base class.
```

Reply:
For my understanding, is there a reason why Godot virtual methods are straying from the general definition? Since users might ask themselves that, could be good to give a warning here

Reply:
Unfortunately, I don't know! Virtual methods in Godot have worked this way longer than I've been a Godot contributor :-)

However, in the context of this release page, I think just giving a simpler explanation with fewer details would be sufficient.